governance/polls/Proposal - Adjust the BAT Risk Premium

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Make sure the filename has the .md suffix.

Reply:
BAT-A rather than BAT.

Reply:
👍